pangea-sdk/service/authn/api.go

```diff
@@ -1219,48 +1346,48 @@ const (
 	SLOBactiveTokenID                    = "active_token_id"
 )
 
-type SessionListOrder string
+type ItemOrder string
 
 const (
-	SLOasc  SessionListOrder = "asc"
-	SLOdesc                  = "desc"
+	IOasc  ItemOrder = "asc"
+	IOdesc           = "desc"
 )
 
 type ClientSessionListRequest struct {
 	Token   string             `json:"token"`
 	Filter  map[string]string  `json:"filter,omitempty"`
 	Last    string             `json:"last,omitempty"`
-	Order   SessionListOrder   `json:"order,omitempty"`
+	Order   ItemOrder          `json:"order,omitempty"`
 	OrderBy SessionListOrderBy `json:"order_by,omitempty"`
 }
 
 type SessionToken struct {
-	ID        string            `json:"id"`
-	Type      string            `json:"type"`
-	Life      int               `json:"list"`
-	Expire    string            `json:"expire"`
-	Email     string            `json:"email"`
-	Scopes    []string          `json:"scopes"`
-	Profile   map[string]string `json:"profile"`
-	CreatedAt string            `json:"created_at"`
+	ID        string      `json:"id"`
+	Type      string      `json:"type"`
+	Life      int         `json:"list"`
+	Expire    string      `json:"expire"`
+	Email     string      `json:"email"`
+	Scopes    Scopes      `json:"scopes"`
+	Profile   ProfileData `json:"profile"`
+	CreatedAt string      `json:"created_at"`
 }
 
 type SessionItem struct {
-	ID          string            `json:"id"`
-	Type        string            `json:"type"`
-	Life        int               `json:"list"`
-	Expire      string            `json:"expire"`
-	Identity    string            `json:"identity"`
-	Email       string            `json:"email"`
-	Scopes      []string          `json:"scopes"`
-	Profile     map[string]string `json:"profile"`
-	CreatedAt   string            `json:"created_at"`
-	ActiveToken SessionToken      `json:"active_token"`
-	Last        string            `json:"last"`
+	ID          string       `json:"id"`
+	Type        string       `json:"type"`
+	Life        int          `json:"list"`
+	Expire      string       `json:"expire"`
+	Identity    string       `json:"identity"`
+	Email       string       `json:"email"`
+	Scopes      Scopes       `json:"scopes"`
+	Profile     ProfileData  `json:"profile"`
+	CreatedAt   string       `json:"created_at"`
+	ActiveToken SessionToken `json:"active_token"`
```

Comment:
active_token is optional, might be omitted.  this should probably be `*SessionToken`